packages/workflow/src/constants.ts

```diff
@@ -24,8 +24,8 @@ export const METRICS_URL = process.env.METRICS_URL || 'http://localhost:1050/'
 
 export const MOSIP_TOKEN_SEEDER_URL =
   process.env.MOSIP_TOKEN_SEEDER_URL || 'http://localhost:8085'
-export const RESOURCE_SERVICE_URL =
-  process.env.RESOURCE_SERVICE_URL || `http://localhost:3040/`
+export const COUNTRY_CONFIG_URL =
+  process.env.COUNTRY_CONFIG_URL || `http://localhost:3040/`
```

Comment:
The COUNRY_CONFIG_URL declared here: https://github.com/opencrvs/opencrvs-core/pull/6448/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R77 doesn't have the trailing slash. We should probably keep them in sync